src/components/forms/formr-part-b/View.tsx

```diff
@@ -864,6 +863,10 @@ const View = ({ canEdit, history }: IView) => {
                 </SummaryList>
               </Card.Content>
             </Card>
+            <h3>
+              Form Submitted on:&nbsp;
+              {DateUtilities.ToUTCDate(formData.submissionDate)}
```

Comment:
Looks good, @EdwardBarclay ! 
Do you think it's worth adding a small test for the submission date visibility? (e.g. formr-part-a/View.cy-test.tsx ) 

Just had a look at the ticket...I know you have followed the format given (UTC).  Just wondering if it would be better to keep things consistent and use local date format? Just a thought.